include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -95,18 +99,22 @@ class ADIOS2IOHandlerImpl
 
 #if openPMD_HAVE_MPI
 
-    ADIOS2IOHandlerImpl( AbstractIOHandler *, MPI_Comm );
+    ADIOS2IOHandlerImpl( AbstractIOHandler *, MPI_Comm, nlohmann::json config );
 
     MPI_Comm m_comm;
 
 #endif // openPMD_HAVE_MPI
 
-    explicit ADIOS2IOHandlerImpl( AbstractIOHandler * );
+    explicit ADIOS2IOHandlerImpl(
+        AbstractIOHandler *
+        , nlohmann::json config
+    );
 
 
-    ~ADIOS2IOHandlerImpl( ) override = default;
+    ~ADIOS2IOHandlerImpl() override = default;
 
-    std::future< void > flush( ) override;
+    std::future< void >
+    flush() override;
```

Comment:
this block has unnecessary whitespace changes and somehow an odd linebreak in the constructor parameters